crates/typst-syntax/src/kind.rs

```diff
@@ -490,6 +496,8 @@ impl SyntaxKind {
             Self::FuncReturn => "`return` expression",
             Self::Destructuring => "destructuring pattern",
             Self::DestructAssignment => "destructuring assignment expression",
+            Self::Write18 => "write18 command",
+            Self::InputPipe => "write18 command",
```

Comment:
this should be in italian

```suggestion
            Self::Write18 => "comando write18",
            Self::InputPipe => "comando write18",
```

Reply:
Non non non, nous devons honorer La République! Ca doit être en français! `commande write18`

Reply:
Ich werde mich dieser Diskussion entziehen, bis ein Ergebnis gefunden wurde. Das kann doch wohl nicht so schwer sein.

Reply:
do'o jai fenki .i li'a prane fa lu la'oi write18 noi minde li'u noi lojbo